101-machine-learning-service-create/azuredeploy.json

```diff
@@ -0,0 +1,288 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "webserviceName":{
```

Comment:
add metadata descriptions to all parameters

Reply:
addeed